src/js/app.js

```
@@ -522,18 +527,22 @@ function applyFilters(){
   if(cat)AN.trackCat(cat);
 
-  let res=ORGS.filter(o=>{
+  const res=ORGS.filter(o=>{
     const txt=(o.name+' '+o.tags.join(' ')+' '+o.desc).toLowerCase();
     if(cat&&o.cat!==cat)return false;
     if(lang&&!txt.includes(lang))return false;
     if(search&&!txt.includes(search))return false;
-    if(yearsF){const yc=yCls(o.years);if(yearsF!==yc)return false}
+    if(yearsF){const yc=yCls(o.years);if(yearsF!==yc)return false;}
     if(compF&&o.competition!==compF)return false;
+HEAD
```

ESLint

[error] 'HEAD' is not defined.

(no-undef)